parlai/mturk/core/mturk_manager.py

```diff
@@ -1859,3 +1862,186 @@ def email_worker(self, worker_id, subject, message_text):
             return {'failure': failure_message['NotifyWorkersFailureMessage']}
         else:
             return {'success': True}
+
+
+# TODO consolidate base functionality out of this class and above into a
+# base_crowd_manager and then expand out from there.
+class StaticMTurkManager(MTurkManager):
+    """Manages interactions between MTurk agents and tasks, the task launching
+    workflow, and more, but only for tasks that require just 2 connections
+    to the server: an initial task request and the submission of results
+    """
+
+    def __init__(self, opt, is_test=False):
+        """No interaction means only ever one agent, so that's what we get"""
+        opt['max_connections'] = 0  # Max connections doesn't make sense here
+        opt['count_complete'] = True  # No other way to count static HITs
+        opt['frontend_template_type'] = 'static'
+        super().__init__(opt, ['worker'], is_test, True)
+        self.hit_mult = 1  # No need to pad HITs if they're static
+        base_required_hits = self.num_conversations * len(self.mturk_agent_ids)
```

Comment:
assuming this will always be equal to `self.num_conversations` as `len(self.mturk_agent_ids) == 1`

Reply:
True - this was copypasta